packages/twenty-ui/src/display/chip/components/Chip.tsx

```diff
@@ -111,6 +111,10 @@ const StyledContainer = withTheme(styled.div<
 
   border-radius: ${({ theme, variant }) =>
     variant === ChipVariant.Rounded ? '50px' : theme.border.radius.sm};
+
+  &.relation-chip {
+    height: ${({ theme }) => theme.spacing(3)};
```

Comment:
Let's just modify the Chip component to `height: ${({ theme }) => theme.spacing(4)};` for all use cases.

Also for information, this pattern you used, using a class to modify the default behavior is not standard in Twenty, we prefer adding a prop like 'small' | 'big' and then convert this into the relevant CSS. 

But since here we just want to modify the Chip component we don't need it.

Reply:
Thanks, making changes, In Issue it was mentioned just relation chips that made me do changes conditionally as Chip Component is being used beyond that.